buildSrc/build.gradle.kts

```diff
@@ -4,13 +4,15 @@ plugins {
 
 repositories {
     jcenter()
-    gradlePluginPortal()
+    maven {
+        url = uri("https://plugins.gradle.org/m2/")
+    }
 }
 
 dependencies {
     implementation("org.springframework.boot:spring-boot-gradle-plugin:2.4.2")
     implementation("org.cqfn.diktat:diktat-gradle-plugin:0.4.0")
     implementation("io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.15.0")
     implementation("org.ajoberstar.reckon:reckon-gradle:0.13.0")
-    implementation("org.jetbrains.kotlin.plugin.allopen:org.jetbrains.kotlin.plugin.allopen.gradle.plugin:1.3.71")
+    implementation("org.jetbrains.kotlin:kotlin-allopen:1.4.30-RC")
```

Comment:
Please use same kotlin version as in `Versions.kt`. I'm afraid there is no way to reuse it, so just change it here.

Reply:
Remade to 1.4.21